### ISSUE TITLE
ドラDORAジャンケン (Dora DORA Janken) - Very minor audio regression

### ISSUE THREAD

**Issue by sombraguerrero** [ruffle-rs/ruffle#18057]
### Describe the bug

As of the July 8th 2024 nightly builds, tested on both web and desktop, there is the slightest regression that has occurred in this game where the sound effect on game-start does not play.
The only output from the logs via the desktop app that might be relevant are the warnings from symphonia about overflow, underflow, and invalid starting and end positions in the byte streams, but those are warnings I've always seen and had no issues.
Yes, this is an old version 5 SWF.

### Expected behavior

Transition sound should play on game start

### Content Location

[djk_0329.zip](https://github.com/user-attachments/files/17108474/djk_0329.zip)

### Affected platform

Desktop app

### Operating system

Windows 11

### Browser

Chromium Edge

### Additional information

2024-09-24T05:14:11.954312Z  WARN symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 91 bytes    
2024-09-24T05:14:13.996396Z  WARN run_frame:run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:14.167027Z  WARN symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:14.337020Z  WARN symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:14.507021Z  WARN symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:14.677043Z  WARN symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:16.166683Z  WARN run_frame:run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:16.964324Z  WARN run_frame:run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:19.230101Z  WARN run_frame:run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:20.542982Z  WARN run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes    
2024-09-24T05:14:22.309447Z  WARN run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 91 bytes    
2024-09-24T05:14:23.142786Z  WARN run_frame:run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 91 bytes    
2024-09-24T05:14:24.676003Z  WARN run_frame:run_frame: symphonia_bundle_mp3::layer3: mpa: invalid main_data_begin, underflow by 65 bytes

**Comment by sombraguerrero** [comment]
I'm just going to guess that #17016 is probably the culprit. Just would make sense.

**Comment by ChrisCPI** [comment]
Bumping this, as I was about to open an issue related to this kind of thing. I also suspected that #17016 was the cause of this.

Specifically, in Super Mario 63, there are several instances where audio behavior isn't quite right. A couple of examples are:
- In the popup telling you something about some kind of milestone you've reached with the shine sprite count (e.g. another room in the castle has been opened), and the jingle plays, it gets cut off when the popup is closed. In Flash Player, the jingle continues to play for its duration after the popup has been closed.
- The sound effect when equipping a FLUDD from a pod does not seem to play at all.
